src/contour/contour.yml

```diff
@@ -64,6 +64,9 @@ pty_buffer_size: 1048576
 
 default_profile: main
 
+# Time in seconds to check for early threshold
+early_exit_threshold: 5
+
```

Comment:
Didn't we make this contour.yml file obsolete?

Reply:
We do not use it to generate config file, this is true, i just placed it there to keep it updated until we have a better way to see full default config file